app/forms/benefit_details_form.rb

```diff
@@ -25,6 +25,9 @@ def benefit_list
     jobseekers_allowance
     employment_support_allowance
     pension_credit
+    guaranteed_min_pension
```

Comment:
As these aren't passported benefits, let's rename `PASSPORTED_BENEFITS` to something more relevant, like `BENEFITS_TO_OMIT` or something